### PR TITLE
create reports and project for the update test

### PR DESCRIPTION
**Pull request by gyorb**
with the new generated report files no analysis
is required during the tests

source file path needs to be modified in the report files
to point to the right source files

### CODE REVIEW

File: web/tests/functional/update/test_proj/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Makfile to build and analyze the test files required
```
Comment (csordasmarton) [inline web/tests/functional/update/test_proj/Makefile]:
```suggestion
# Makefile to build and analyze the test files required
```

File: web/tests/functional/update/__init__.py [hunk 2]
```diff
@@ -31,33 +35,30 @@ def setup_package():
...
+    test_project_name = uuid.uuid4().hex
 
-    test_project_name = project_info['name'] + '_' + uuid.uuid4().hex
+    test_project_path = os.path.join(test_dir, "test_proj")
 
-    test_config['test_project'] = project_info
+    temp_test_project_data = project.prepare(test_project_path, TEST_WORKSPACE)
 
-    suppress_file = None
+    print(temp_test_project_data)
```
Comment (csordasmarton) [inline web/tests/functional/update/__init__.py]: Do we want to keep this? This was only for debug purposes.

File: web/tests/functional/update/test_proj/Makefile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# for the store update tests.
+
+build:
+	$(CXX) $(currentdir)/main.cpp -o main
```
Comment (csordasmarton) [inline web/tests/functional/update/test_proj/Makefile]: What is `currentdir`? Where do you declare it?

**Comment by gyorb** [comment]
I've extended the documentation, please check if it is enough.

**Pull request merged**